app/views/public_recipes/index.html.erb

```diff
@@ -0,0 +1,14 @@
+<h1>Recipes</h1>
+<% @recipes.each do |recipe| %>
+<a href="recipes/<%= recipe.id %>">Recipe #<%= recipe.id %></a>
+<div>Name: <%= recipe.name %> by <%= recipe.user.name %></div>
+<div>Description: <%= recipe.description %></div>
+<div>Public? <%= recipe.public %></div>
+<% RecipeFood.where(recipe: recipe).each do | recipe_food | %>
```

Comment:
- [OPTIONAL] Kindly fix the N + 1 queries issue adding `.includes(:food)` to the query. It's a project requirement.